CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ Please only add new entries below the [Unreleased](#unreleased---releasedate) he
 
 ## [@Unreleased] - @ReleaseDate
 
+### Features
+
+- Support the overlay (@wjian23)
```

Comment:
Introduced the Overlay Feature: This enhancement simplifies the creation of overlay widgets. It streamlines the addition of any widget to an overlay and offers a more user-friendly API for overlay management